config/7.0/obdemo-bank/ig/scripts/groovy/RepoConsent.groovy

```diff
@@ -140,10 +149,10 @@ enum IntentType {
     ACCOUNT_ACCESS_CONSENT("AAC_", "accountAccessIntent"),
     PAYMENT_DOMESTIC_CONSENT("PDC_","domesticPaymentIntent"),
     PAYMENT_DOMESTIC_SCHEDULED_CONSENT("PDSC_", "domesticScheduledPaymentIntent"),
-    PAYMENT_DOMESTIC_STANDING_ORDERS_CONSENT("PDSOC_", "domesticStandingOrdersPaymentIntent"),
+    PAYMENT_DOMESTIC_STANDING_ORDERS_CONSENT("PDSOC_", "domesticStandingOrderPaymentIntent"),
```

Comment:
I think that we need to rename the enum constant from `PAYMENT_DOMESTIC_STANDING_ORDERS_CONSENT` (ORDERS) to `PAYMENT_DOMESTIC_STANDING_ORDER_CONSENT` (ORDER) to be aligned with the object name.

Reply:
We cannot rename the enum variable, because the value `PAYMENT_DOMESTIC_STANDING_ORDERS_CONSENT` is used in rcs with the same format. https://github.com/SecureBankingAccessToolkit/securebanking-openbanking-uk-rcs/blob/master/securebanking-forgerock-cloud-client/src/main/java/com/forgerock/securebanking/platform/client/IntentType.java

Reply:
`domesticStandingOrderPaymentIntent` is the name of the managed object created in IDM.